### DISCUSSION TITLE
[Enhancement] [image-viewer] image-viewer 是否可以支持自定义底部按钮

### DISCUSSION THREAD

**Discussion by Lzy328** [element-plus/element-plus#16724] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `image-viewer`

## Description

希望可以支持自定义操作按钮

### Additional comments

_(empty)_

**Comment by zouyaoji** [comment]
+1

但看了源码 应该是不支持，因为没留插槽。

**Comment by zzjiaxiang** [comment]
#19497

**Comment by simlucky** [comment]
+1

**Comment by warmthsea** [comment]
2.9.4版本开始已经支持
